packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -775,12 +803,14 @@ suite('lit-html', () => {
         '<div foo="A"></div>',
         'A'
       );
+      const observer = new MutationObserver(() => {});
```

Comment:
eek 😬

Reply:
Since the attribute is bound to div in the container, not the container yourself, I think you need to observe `container.firstChild`, or else `{attributes: true, subtree: true}`? I guess the latter is probably better so you catch anything.